src/Google.Logging.Log4Net/GoogleStackdriverAppender.cs

```diff
@@ -40,7 +40,12 @@ public sealed partial class GoogleStackdriverAppender : AppenderSkeleton
         // TODO:
         // * Various argument validations
         // * Send unsent logs on program exit? Using AppDomain.ProcessExit
-        //   - Note this only allows 2 seconds 
+        //   - Note this only allows 2 seconds
+
+        // For testing only.
```

Comment:
I'd still get rid of these, personally. I don't think it matters that we don't check the output - a comment at the end of the snippet would be fine.